apps/ui/src/components/SpaceSettingsAlerts.vue

```diff
@@ -119,6 +130,12 @@ const hasAnyAlerts = computed(
         </AppLink>
       </UiAlert>
     </template>
+    <template v-if="hasAuthenticatorsAlerts">
+      <UiAlert v-if="isRelayerBalanceLow" type="error">
+        Your relayer balance is low. Ensure sufficient funds to maintain gasless
```

Comment:
Can we use this sentence instead:

"Your relayer balance is running low. Please top up to keep gasless voting active."